packages/fedify/src/webfinger/handler.ts

```diff
@@ -223,6 +230,16 @@ async function handleWebFingerInternal<TContextData>(
     if (image.mediaType != null) link.type = image.mediaType;
     links.push(link);
   }
+
+  if (webFingerLinksDispatcher != null) {
+    const customLinks = await webFingerLinksDispatcher(context, context.url);
```

Comment:
We need to pass the `resourceUrl` here so that a `WebFingerLinksDispatcher` could behave depending on the given `resource`.

```suggestion
    const customLinks = await webFingerLinksDispatcher(context, resourceUrl);
```